### DISCUSSION TITLE
VPS Recommendation

### DISCUSSION THREAD

**Discussion by Red-Bash** [telekom-security/tpotce#1522] [Q&A] [answered]
Good morning:
I would like recommendations for choosing a vps to install T-Pot. I would like to know if anyone has been able to install one that you recommend and that does not present problems.

Thanks

**Comment by Dltd** [comment, marked as answer]
I would say that's not really an issue, but a generic question.
But; totally works on a VPS with 16GB RAM and enough storage, just follow the minimum recommended specs and follow the installation guide.

**Comment by onurhany** [comment]
still doesn't work

**Comment by Dltd** [comment]
@samddo   On Debian 11, 6CPU cores and 16GB RAM I have no problems myself.

@onurhany 
I would say that if 'still doesn't work' is what you think is needed to get anybody willing to support you when raising or commenting on an issue, you may not be suited to run this kind of software.

Not that it will guarantee you of any support or answer from the community or the maintainers, if you post your issue with something like: 
```
- I have this OS, these technical specs and have executed the following commands following the installation guide up to step X, I have confirmed network connectivity inbound, outbound from the host.
- The problem I run into is: error message X, log output Y.
- I tried to investigate doing these steps :
- but am clueless what's up with it.
```

You may have better odds that someone is willing to assist you.
